.github/workflows/turbopack-nightly-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Turobpack Nightly Release
+
+on:
+  schedule:
+    - cron: "15 8 * * *" # Run every day at 00:15 PST
+  workflow_dispatch:
+
+jobs:
+  turbopack_nightly:
+    name: Cut nightly Turbopack release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout latest commit that passes CI
+        uses: talentpair/last-green-commit-action@d95cfa8
+        id: green_commit
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Tag nightly
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.1
```

Comment:
I guess you could also use `git tag turbopack...` and `git push --tags` which gives you full freedom in naming tags.